docs/user/administration/users/buddypress-cover-images.md

```diff
@@ -0,0 +1,97 @@
+# BuddyPress Cover Images
+
+BuddyPress allows members and groups to upload and manage their cover images. This feature enhances the visual appeal of profiles and group pages by providing a customizable background image. Here’s how you can manage BuddyPress cover images in your theme.
+
+## Table of Contents
+
+1. [Introduction](#introduction)
+2. [Default Cover Image Sizes](#default-cover-image-sizes)
+3. [Changing Cover Image Sizes](#changing-cover-image-sizes)
+4. [Cover Image Uploading](#cover-image-uploading)
+5. [Overriding Template Files](#overriding-template-files)
+6. [Customizing with CSS](#customizing-with-css)
+7. [Frequently Asked Questions](#frequently-asked-questions)
+
+## Introduction
+
+BuddyPress cover images allow users to personalize their profiles and groups by adding a background image. This feature is similar to cover images on social media platforms like Facebook and Twitter. By default, cover images are enabled for both members and groups.
+
+## Default Cover Image Sizes
+
+By default, BuddyPress sets the cover image size to 1300px wide by 225px tall. These dimensions ensure that cover images look good on a variety of devices and screen sizes.
```

Comment:
```suggestion
Just like it does it can be a good idea for your Theme Compat feature to include extra filters into your code to let advanced users customize it to better match their theme. Below are examples about how it's possible to modify default cover image settings or completely disable the feature.
```